app/main/views/stats.py

```diff
@@ -24,13 +24,14 @@ def view_statistics(framework_slug):
             audit_type=AuditTypes.snapshot_framework_stats,
             per_page=1260
         )['auditEvents']
-        latest = data_api_client.get_framework_stats(framework_slug)
-        snapshots.append(
-            {
-                'data': latest,
-                'createdAt': datetime.utcnow().strftime(DATETIME_FORMAT)
-            }
-        )
+        framework = data_api_client.get_framework(framework_slug)['frameworks']
+        if framework['status'] is 'open':
```

Comment:
We should use `==` for comparing values of things (strings) - `is` tests for identity and seems like it is only generally used for `is None`.  See http://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce

Otherwise good to go I think.